cmd/common/st/check.go

```diff
@@ -0,0 +1,40 @@
+package st
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/wal-g/tracelog"
+	"github.com/wal-g/wal-g/internal"
+	"github.com/wal-g/wal-g/internal/storagetools"
+)
+
+var checkCmd = &cobra.Command{
+	Use:   "check",
+	Short: "check access to the storage",
+}
+
+var checkReadCmd = &cobra.Command{
+	Use:   "read",
+	Short: "check read access to the storage",
+	Args:  cobra.MinimumNArgs(0),
```

Comment:
`cobra.MinimumNArgs(1)`?

Reply:
no, checking particular files is optional. 
we may check that storage is readable without any particular files